7.2 Use com.mysql.cj.jdbc.Driver

jhinkey opened the pull request:
Update the driver and add serverTimezone=GMT to the mysql URLs.

Includes Wildfly fix to keep Weld config https://issues.liferay.com/browse/LRDOCS-7898

sez11a commented:
@jhinkey Can you genericize these instead? This is not the place to document MySQL URLs.

jhinkey commented:
Yep, I'll do that.

jhinkey commented:
I genericized the database content. 

Also I updated these things to be in sync with learn.liferay:
1. Removed/replaced listings of ZIP JARs in installer docs
2. Removed/replaced several installer checkpoints
3. Replaced the shard migration content wholesale
4. Replaced the Wildfly JNDI DB configuration
